gui/curs_lib.c

```diff
@@ -135,6 +135,9 @@ void mutt_refresh(void)
  */
 void mutt_need_hard_redraw(void)
 {
+  // Forcibly switch to the alternate screen.
+  // Using encryption can leave ncurses confused about which mode it's in.
+  fputs("\033[?1049h", stderr);
```

Comment:
Why stderr?

Reply:
good question :-)
With lots of redirections going on during pin-entry, I wanted to ensure that it was actually emitted.

My slight worry is that there's a CLI code path that can get here that leaves the user in the alternate screen.

Reply:
meaning it would be bogus to run neomutt with stderr redirected to file anyway?

Reply:
updated: `stdout`, `fflush()`